src/app/components/user-details/user-details.component.scss

```diff
@@ -0,0 +1,58 @@
+@import '../../variables.scss';
+
+.account-icon{
+    font-size: 50px;
+}
+
+.mat-card-actions{
+    align-self: flex-end;
+}
+
+.spacer{
+    flex: 1 1 auto;
+}
+
+.mat-divider{
+    margin-bottom: 30px;
+}
+
+.about-view{
+    display: flex;
```

Comment:
You can use the **flex-container** mixin defined in the mixins.scss file.

For example on how to use it see: src\app\components\create-edit-event-page\create-edit-event-page.component.scss